### DISCUSSION TITLE
Question regarding the drone

### DISCUSSION THREAD

**Discussion by nathmo** [jyjblrd/Mocap-Drones#2] [General]
Hi, I'm curious about the drone showed in the video. What is it ? What's the brand / where did you find it ? Thanks a lot :)

**Comment by jyjblrd** [comment]
Edit: [blog post with more details](https://joshuabird.com/blog/post/mocap-drones)

Hey! The drone is built from off the shelf parts to keep the cost down, since all the research-focused drones I saw online were pretty expensive. 

| device | price | name |
|---|---|---|
| Flight controller | $23 | [F3 EVO Micro Brushed Flight Controller running Betaflight](https://www.aliexpress.com/item/1005001674141725.html?spm=a2g0o.order_list.order_list_main.75.75981802k9oeqz) |
| Frame Motor & Props | $13 | [Generic 95mm brushed drone frame, 8520 brushed motors, 55mm props](https://www.aliexpress.com/item/32776829828.html?spm=a2g0o.order_list.order_list_main.80.75981802k9oeqz) |
| Battery | $5 | [YDL 18350 (1100mah, 9A)](https://www.fogstar.co.uk/collections/batteries/products/ydl-18350-battery) |
| Battery holder | $0.5 | [1S 18350 battery holder](https://www.aliexpress.com/item/1005001346705221.html?spm=a2g0o.productlist.main.1.66a540c6FlUSTy&algo_pvid=17e8210c-ba1c-49ba-85af-6d0f9cb2072b&algo_exp_id=17e8210c-ba1c-49ba-85af-6d0f9cb2072b-0&pdp_npi=4%40dis%21GBP%210.73%210.58%21%21%210.88%21%21%40211b442017004054249571715e7418%2112000015769690898%21sea%21UK%213408833611%21&curPageLogUid=mzg5RUzqTg48) |
| IR markers | $0.1 | 3mm IR Led (you need to sandpaper the surface so they emit light evenly, instead of just up) |
| ESP32 receiver | $4 | [WEMOS D1 Mini ESP32 USB-C](https://www.aliexpress.com/item/32858054775.html?spm=a2g0o.order_list.order_list_main.10.75981802k9oeqz) |

Total cost: ~$50 USD
Flight time: ~10min
Thrust/Weight: very shit lol

You should be able to use any drone you want with this system, since the ESP32 emulates an RC receiver sending joystick commands.

At some point I'll write a blog post about some of the details like how to create the cameras and drones. But in the meantime I hope this helps!

**Reply by nathmo** [reply]
I really liked the low-cost approach at making a drone and was curious on where to find the design ^^
thanks for the BOM. Maybe I will take a shot at designing my own. It's been in my TODO's for a while :)

**Reply by FPSychotic** [reply]
Interested in the cameras setup and a IR filter (floppy disk) alternative . Also they go slightly out of focus when the IR blocker is removed. Do they need the cases? i have 4, but no cases. Basically the camera setup.

**Reply by jyjblrd** [reply]
Yeah @FPSychotic I also need to write in my blog post about the cameras, since there is a bit of nuance on which version of PS3 eye cameras can be used, how to convert them to IR and how to adjust the focus for the IR wavelength.

I'll try get it out in the next few days and link it here, but there are also other online resources about converting PS3 eye cameras to IR if you are interested.

**Reply by randy7771026** [reply]
Very old but very new to these project parts.  Too many crazy questions.  Batteries Not Included.  Links to places to buy parts would be fantastic.

**Reply by CleisonManriqueAguirre** [reply]
How to setup the infrared camera ? what do I to build that? Amzing project

**Reply by nathmo** [reply]
battery : 
* https://fr.aliexpress.com/item/4000983389825.html
* https://www.amazon.com/RADIOMASTER-Zorro-18350-900mAh-Battery/dp/B0B7NT1PQJ/ref=sr_1_2?keywords=YDL+18350&qid=1700400957&sr=8-2

18350 is the format of the cell, (18mm x 35 mm)

**Reply by jyjblrd** [reply]
I just added links to where i bought everything. No promises that the links will still be working in a years time tho! The name of the product should be enough to help you find everything

**Reply by CleisonManriqueAguirre** [reply]
But talking about the cameras ? ps3 cameras are around 45 $ . Isnt it ?

**Reply by jyjblrd** [reply]
I bought the ps3 eye cameras [here](https://uk.webuy.com/product-detail?id=spseyee001&categoryName=playstation3-accessories&superCatName=gaming&title=&queryID=75ce5d199a518b642457aa3877fc7417&position=1) for 1 pound (the price has now gone up to 1.50). They are also really cheap second hand on ebay. I'll give more details on the camera in my blog post later.

**Comment by jyjblrd** [comment]
Blog post with more info on the drones & cameras is up!
[https://joshuabird.com/blog/post/mocap-drones](https://joshuabird.com/blog/post/mocap-drones)

**Reply by Flenser42** [reply]
Hey, do you have a link for the lenses for the PS Eye Cameras? BTW. Great work have been waiting for the Blog post since the Video is out lol

**Comment by Jmmx1237** [comment]
Hi,
very cool project, thanks for publishing it.

I wanted to ask, would it also be possible to do the triangulation and positioning using WLAN or small radio transmitters?

The control could be further refined using ML...?
